### ISSUE TITLE
Nextcloud - consider setting default permissions to support WebUI update

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#1182]
I just tried to do an in browser update of Nextcloud and hit this:

![screenshot from 2018-08-23 10-22-37](https://user-images.githubusercontent.com/818633/44497769-86c73780-a6be-11e8-8d76-ea1397ed22f1.png)

IMO we should support the easy in browser upgrade option.

The manual upgrade process is documented [here](https://docs.nextcloud.com/server/13/admin_manual/maintenance/manual_upgrade.html) but I'll document the process to enable WebUI upgrade here shortly.

**Comment by JedMeister** [comment]
To enable WebUI updater, or use commandline updater, first give ownership of Nextcloud directory to webserver (www-data):
```
chown -R www-data:www-data /var/www/nextcloud/
```
Then run through the in-browser upgrade process.

Alternatively, update via commandline:
```
su - www-data -s /bin/bash
cd /var/www/nextcloud/
php occ upgrade
```

[Optional] lock permissions back down:
```
chown -R root:root /var/www/nextcloud
chown -R www-data:www-data /var/www/nextcloud/{config,apps,data}
```

**Comment by SamSirry** [comment]
@JedMeister, I wonder why you closed this issue. It is still not resolved in Mar 2021.

**Comment by JedMeister** [comment]
@SamSirry - Issues are closed when code that resolves them is merged (into the relevant code repository). The code that fixes this particular issue was merged 10 Feb (in [this pull request](https://github.com/turnkeylinux-apps/nextcloud/pull/14)), hence why this one is closed.

If you note the milestone (in this case v16.1), that is the version of the appliance where it is fixed. As you can see from the [Nextcloud appliance page](https://www.turnkeylinux.org/nextcloud) the most recent released version is v16.0. So the fixed version has not yet been released/published.

As a general rule, once new code is merged, we try to release the updated appliance ASAP. But it's not always how things go. In the case of Nextcloud, there were some other tweaks that needed to be done; plus there were also some other changes that I wanted to make consistent with the ownCloud appliance, so I'm planning to release both v16.1 releases of these 2 together.
